librz/util/print.c

```diff
@@ -970,13 +993,13 @@ RZ_API void rz_print_hexdump(RzPrint *p, ut64 addr, const ut8 *buf, int len, int
 					ut8 ch = (use_unalloc && p && !p->iob.is_valid_offset(p->iob.io, addr + j, false))
 						? ' '
 						: buf[j];
-					rz_print_byte(p, "%c", j, ch);
+					sb_print_own(rz_print_byte_str(p, "%c", j, ch));
```

Comment:
`sb_print_own()` used exactly twice. I think it's better to unroll the macro and use these three lines here directly.